automation/check-patch.e2e-kubemacpool-functests.sh

```diff
@@ -9,6 +9,23 @@
 # yum -y install automation/check-patch.packages
 # automation/check-patch.e2e-kubemacpool-functests.sh
 
+function __get_skipped_tests() {
+	local  __resultvar=$1
+	local skipped_regex=""
+
+	# We can't test all KMP opt-mode CNAO context, as the operator will reconcile
```

Comment:
Hey what about this comment, I think you cant understand form this that KMP functests configures the opt-mode
could you change it to something like `check KMP webhook's opt-mode and skip tests accordingly`?

Reply:
added your comment below mine.
Done

Reply:
Thanks